fakespans/README.md

```diff
@@ -0,0 +1,49 @@
+## Creating test data in kafka 
+
+fakespans is a simple go app which can generate random spans and push to kafka 
+
+## Using fakespans
+
+Run the following commands on your terminal to start using fake spans you should have golang installed on your box
+
+1. export $GOPATH=`location where you want your go binaries`
+2. export $GOBIN=$GOPATH/bin
+3. cd fakespans
```

Comment:
do we need to add go get before, before go install ?

Reply:
Added `go get` to the document now